superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -417,6 +422,54 @@ class SqlEditor extends React.PureComponent {
     );
   }
 
+  renderDropdown() {
+    const qe = this.props.queryEditor;
```

Comment:
I've been changing this in this file when I find it to use deconstruction instead.